### PR TITLE
ARUHA-802 Update lastCommitTime for subscription

### PR DESCRIPTION
**Pull request by antban**
when stream switches from state where everything was committed to state with something is uncommitted.

### CODE REVIEW

**Comment by antban** [comment]
:+1:

File: src/main/java/org/zalando/nakadi/service/subscription/state/StreamingState.java [hunk 1]
```diff
@@ -192,12 +203,15 @@ private void checkBatchTimeouts() {
     private void streamToOutput() {
         final long currentTimeMillis = System.currentTimeMillis();
         int messagesAllowedToSend = (int) getMessagesAllowedToSend();
+        final boolean wasCommitted = isEverythingCommitted();
```
Comment (adyach) [inline src/main/java/org/zalando/nakadi/service/subscription/state/StreamingState.java]: is it the same as `messagesAllowedToSend == 0` ?

Comment (antban) [inline src/main/java/org/zalando/nakadi/service/subscription/state/StreamingState.java]: no, it's not messagesAllowedToSend is > 0 even if something was uncommitted

**Comment by adyach** [comment]
👍

**Pull request merged**